src/Prolens.hs

```diff
@@ -272,6 +286,39 @@ The other element of 'Either' (variable of type @c@) can be of any
 type, and you can decide what type it should be. This is convenient
 for implementing various functions. E.g. 'prism' uses this fact.
 
+
+Assuming, we have the following functions in scope:
+
+@
+swapEither :: Either a b -> Either b a
+swapEither (Left a) = Right a
+swapEither (Right b) = Left b
+
+unnestLeft :: Either (Either a b) c -> Either a (Either b c)
+unnestLeft (Left (Left a)) = Left a
+unnestLeft (Left (Right b)) = Right (Left b)
+unnestLeft (Right c) = Right (Right c)
+
+nestLeft :: Either a (Either b c) -> Either (Either a b) c
+nestLeft (Left a) = Left (Left a)
+nestLeft (Right (Left b)) = Left (Right b)
+nestLeft (Right (Right c)) = Right c
```

Comment:
I think for docs it is okay to just leave types of these functions. They are very clear in what the functions do 👍🏼 🔮